src/components/App.js

```diff
@@ -1,77 +1,75 @@
-import {
-  bookChainContract,
-  accounts
-} from '../ethereum/EthereumClient';
-import React, {
-  Component
-} from 'react';
-import logo from '../logo.svg';
-import ContractForm from './ContractForm';
-import BookForm from './BookForm.js';
-import Carousel from './Carousel';
-import request from 'superagent';
-import _ from 'lodash';
-import '../css/App.css';
-const API = require('./config.json');
+import {Bookchain, accounts} from '../ethereum/EthereumClient'
+import React, { Component } from 'react'
+import logo from '../logo.svg'
+import ContractForm from './ContractForm'
+import BookForm from './BookForm.js'
+import Carousel from './Carousel'
+import request from 'superagent'
+import _ from 'lodash'
+import '../css/App.css'
+
+
 
 class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      bookchainContract: '',
-      books: [{
-        title: "Don Quixote",
-        author: "Miguel de Cervantes",
-        desc: "Test book!",
-        id: "Test1"
-      }]
+      bookchainContract: "",
+      books: [
+        {
+          title: "Don Quixote",
+          author: "Miguel de Cervantes",
+          desc: "Test book!",
+          id: "Test1"
+        }
+      ]
     };
     this.addBook = this.addBook.bind(this);
     this.addBookToBookchain = this.addBookToBookchain.bind(this);
   }
 
-  addBookToBookchain(isbn, bookData, accessibilityData) {
-    bookChainContract.createBook(isbn, {
-      from: accounts[0],
-      gas: 1000000
-    });
-    this.addBook(bookData, accessibilityData);
+  addBookToBookchain(isbn, bookData) {
+    let contract = this.state.bookchainContract
+    Bookchain.at(contract).createBook(isbn, {from: accounts[0], gas: 1000000})
+    this.addBook(bookData)
   }
+  
+  addBook(book) {
+      this.setState({
+        books: this.state.books.concat({
+          title: book.title,
+          author: book.authors[0],
+          id: book.industryIdentifiers[0].identifier,
+          desc: book.description,
+          img_url: book.imageLinks.smallThumbnail
+        })
 
-  addBook(book, accessibilityData) {
-    console.log(book);
-
-    this.setState({
-      books: this.state.books.concat({
-        title: book.title,
-        author: book.authors[0],
-        id: book.industryIdentifiers[0].identifier,
-        desc: book.description,
-        img_url: book.imageLinks.smallThumbnail,
-        accessibile: accessibilityData
       })
```

Comment:
@Sh1pley Will your PR remove the accessibility code I added? Comparing your PR with master, looks like it will. Maybe I am not seeing something?